### PR TITLE
refactor(pack): refactor to use import export

### PR DESCRIPTION
**Pull request by Heiiji**
Signed-off-by: jjuret <julien.juret@gmail.com>

| Question         | Answer
| ---------------- | ---
| Branch?          |perf/telecom
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-6001
| License          | BSD 3-Clause

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/voipBillingAccount/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('PackVoipBillingAccountCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/slots/task/pack-voipBillingAccount.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/voipBillingAccount/index.js:26]:
```suggestion
        'app/telecom/pack/slots/voipBillingAccount/voipBillingAccount.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('PackDomainCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/slots/domain/pack-domain.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/index.js:26]:
```suggestion
        'app/telecom/pack/slots/domain/domain.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/exchangeAccount/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('PackExchangeAccountCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/slots/exchangeAccount/pack-exchangeAccount.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/exchangeAccount/index.js:26]:
```suggestion
        'app/telecom/pack/slots/exchangeAccount/exchangeAccount.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/exchangeLite/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('PackExchangeLiteCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/slots/exchangeLite/pack-exchangeLite.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/exchangeLite/index.js:26]:
```suggestion
        'app/telecom/pack/slots/exchangeLite/exchangeLite.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/voipEcoFax/index.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('PackVoipEcoFaxCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/slots/voipEcoFax/pack-voipEcoFax.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/voipEcoFax/index.js:26]:
```suggestion
        'app/telecom/pack/slots/voipEcoFax/voipEcoFax.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/xdslAccess/index.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('PackAccessCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/slots/xdslAccess/pack-xdslAccess.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/xdslAccess/index.js:26]:
```suggestion
        'app/telecom/pack/slots/xdslAccess/xdslAccess.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/ip/order/index.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslAccessIpOrderCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/access/ip/order/pack-xdsl-access-ip-order.modal.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/ip/order/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/access/ip/order/order.modal.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/ipv6/index.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslAccessIpv6Ctrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/access/ipv6/pack-xdsl-access-ipv6.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/ipv6/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/access/ipv6/ipv6.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/portReset/index.js [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslAccessPortResetCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/access/portReset/pack-xdsl-access-port-reset.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/portReset/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/access/portReset/port-reset.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/profil/index.js [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslAccessProfileCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/access/profil/pack-xdsl-access-profil.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/profil/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/access/profil/profil.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/rateLimit/index.js [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslAccessLnsRateLimitCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/access/rateLimit/xdsl-access-lns-ratelimit.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/rateLimit/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/access/rateLimit/ratelimit.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/bridgeMode/index.js [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemBridgeModeCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/bridgeMode/pack-xdsl-modem-bridgeMode.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/bridgeMode/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/bridgeMode/bridgeMode.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/connectedDevices/index.js [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemConnectedDevicesCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/connectedDevices/pack-xdsl-modem-connectedDevices.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/connectedDevices/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/connectedDevices/connectedDevices.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/firmware/index.js [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemFirmwareCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/firmware/pack-xdsl-modem-firmware.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/firmware/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/firmware/firmware.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/managedByOvh/index.js [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemManagedByCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/managedByOvh/pack-xdsl-modem-managedByOvh.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/managedByOvh/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/managedByOvh/managedByOvh.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/mtu/index.js [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemMtuCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/mtu/pack-xdsl-modem-mtu.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/mtu/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/mtu/mtu.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/reboot/index.js [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemRebootCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/reboot/pack-xdsl-modem-reboot.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/reboot/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/reboot/reboot.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/reset/index.js [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemResetCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/reset/pack-xdsl-modem-reset.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/reset/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/reset/reset.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/dhcp/index.js [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+    'ui.router',
+  ])
+  .controller('XdslModemDhcpCtrl', controller)
+  .controller('telecomXdslModemDhcpModal', controllerModal)
+  .factory('telecomXdslModemDhcpModal', factory)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/router/dhcp/pack-xdsl-modem-dhcp.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/dhcp/index.js:32]:
```suggestion
        'app/telecom/pack/xdsl/modem/router/dhcp/dhcp.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/lan/index.js [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemLanCtrl', controller)
+  .factory('PackXdslModemLanObject', factory)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/router/lan/pack-xdsl-modem-lan.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/lan/index.js:28]:
```suggestion
        'app/telecom/pack/xdsl/modem/router/lan/lan.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/index.js [hunk 21]
```diff
@@ -0,0 +1,34 @@
...
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemPortsCtrl', controller)
+  .factory('PackXdslModemLanObject', factory)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/router/ports/pack-xdsl-modem-ports.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/ports/index.js:28]:
```suggestion
        'app/telecom/pack/xdsl/modem/router/ports/ports.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/service/index.js [hunk 22]
```diff
@@ -0,0 +1,34 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemServiceCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/service/pack-xdsl-modem-service.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/service/index.js:28]:
```suggestion
        'app/telecom/pack/xdsl/modem/service/service.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/service/modal/index.js [hunk 23]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemServiceModalCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/service/modal/pack-xdsl-modem-service-modal.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/service/modal/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/service/modal/service-modal.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/index.js [hunk 24]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemWifiCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/wifi/pack-xdsl-modem-wifi.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/wifi/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/wifi/wifi.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/index.js [hunk 25]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslModemRouterCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/modem/router/pack-xdsl-modem-router.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/modem/router/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/modem/router/router.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/deconsolidation/deconsolidation.controller.js [hunk 26]
```diff
@@ -0,0 +1,82 @@
...
+    validDeconso: false,
+    rio: null,
+  };
+  this.rioCodeIsValid = false;
+
+  this.openConfirmModal = function openConfirmModal() {
+    const modal = $uibModal.open({
+      animation: true,
+      templateUrl:
+        'app/telecom/pack/xdsl/access/deconsolidation/contract/pack-xdsl-access-deconsolidation-contract.modal.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/deconsolidation/deconsolidation.controller.js:28]:
```suggestion
        'app/telecom/pack/xdsl/access/deconsolidation/contract/contract.modal.html',
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/deconsolidation/contract/index.js [hunk 27]
```diff
@@ -0,0 +1,32 @@
...
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .controller('XdslDeconsolidationContractCtrl', controller)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .run(
+    /* @ngInject */ ($templateCache) => {
+      $templateCache.put(
+        'app/telecom/pack/xdsl/access/deconsolidation/contract/pack-xdsl-access-deconsolidation-contract.modal.html',
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/deconsolidation/contract/index.js:26]:
```suggestion
        'app/telecom/pack/xdsl/access/deconsolidation/contract/contract.modal.html',
```

File: packages/manager/apps/telecom/src/app/app.less [hunk 28]
```diff
@@ -104,13 +104,10 @@
 @import './telecom/pack/move/pack-move.less';
 @import './telecom/pack/resiliation/pack-resiliation.less';
 @import './telecom/pack/slots/voipLine/activation/pack-voipLine-activation.less';
-@import './telecom/pack/xdsl/access/deconsolidation/pack-xdsl-access-deconsolidation.less';
-@import './telecom/pack/xdsl/access/linediagnostic/pack-xdsl-access-linediagnostic.less';
+@import './telecom/pack/xdsl/access/linediagnostic/linediagnostic.less';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/app.less:107]: Should be imported in linediagnostic module.

File: packages/manager/apps/telecom/src/app/app.less [hunk 29]
```diff
@@ -104,13 +104,10 @@
...
 @import './telecom/pack/xdsl/access/pack-xdsl-access.less';
 @import './telecom/pack/xdsl/modem/pack-xdsl-modem.less';
-@import './telecom/pack/xdsl/orderFollowUp/pack-xdsl-orderFollowUp.less';
-
-@import './telecom/telephony/telecom-telephony.less';
+@import './telecom/pack/xdsl/orderFollowUp/orderFollowUp.less';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/app.less:110]: Should be imported in orderFollowUp module.

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/task/index.js [hunk 30]
```diff
@@ -0,0 +1,29 @@
...
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import controller from './task.controller';
+import template from './task.html';
+
+const moduleName = 'PackTask';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/task/index.js:11]:
```suggestion
const moduleName = 'ovhManagerTelecomPackSlotsTask';
```

As done in https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/telephony.module.js#L10

File: packages/manager/apps/telecom/src/app/telecom/pack/dashboard/pack.module.js [hunk 31]
```diff
@@ -1,12 +1,32 @@
 import angular from 'angular';
 
 import packMove from '../move';
+import packResiliation from '../resiliation/pack-resiliation';
 import packVoipLineActivation from '../slots/voipLine/activation/pack-voipLine-activation.module';
+import packEcoFaxActivation from '../slots/voipEcoFax/activation';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/dashboard/pack.module.js:6]: Is it possible to have a `slots` module in `../slots` that import children modules  ?

Here all imports could be replaced by `import slots from '../slots';`.

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/emailPro.controller.js [hunk 32]
```diff
@@ -0,0 +1,38 @@
+import map from 'lodash/map';
+
+export default /* @ngInject */ class PackEmailProCtrl {
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/emailPro.controller.js:3]: on `class` `/* @ngInject */` should be added line before `constructor` (example: https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/telephony.controller.js#L3-L5)

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/exchangeIndividual/add/add.controller.js [hunk 33]
```diff
@@ -0,0 +1,90 @@
+import forEach from 'lodash/forEach';
+import get from 'lodash/get';
+
+export default /* @ngInject */ function PackExchangeIndividualEmailAddCtrl(
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/exchangeIndividual/add/add.controller.js:4]: 👍

**Review by cbourgois — COMMENTED** [review]
Nice job 👍

File: packages/manager/apps/telecom/src/app/telecom/pack/dashboard/pack.module.js [hunk 34]
```diff
@@ -1,29 +1,20 @@
...
 
 angular
-  .module(moduleName, [
-    hostedEmailDetail,
-    packVoipLineActivation,
-    xdsl,
-    packMove,
-    migration,
-  ])
+  .module(moduleName, [xdsl, packMove, migration, packResiliation, slots])
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/dashboard/pack.module.js:15]: Is it possible to order in alphabetical order ?

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/index.js [hunk 35]
```diff
@@ -0,0 +1,56 @@
...
+    packExchangeLite,
+    hubic,
+    packInformation,
+    promotionCode,
+    task,
+    voipBillingAccount,
+    voipEcoFax,
+    voipLine,
+    xdslAccess,
+  ])
```
Comment (Steffy29) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/index.js:53]: Alphabetical ordered should be better ;-)

**Comment by frenautvh** [comment]
with this refactor is it possible to use ocLazyLoad for pack section ?

**Comment by Heiiji** [comment]
it is a subject for a other new PR after this one :)

**Review by marie-j — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/add/add.controller.js [hunk 36]
```diff
@@ -0,0 +1,109 @@
+import get from 'lodash/get';
+import map from 'lodash/map';
+
+export default /* @ngInject */ class PackEmailProAddCtrl {
+  constructor(
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/add/add.controller.js:5]:
```suggestion
  /* @ngInject */
  constructor(
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/diagnostic/details/details.controller.js [hunk 37]
```diff
@@ -0,0 +1,58 @@
+import get from 'lodash/get';
+import has from 'lodash/has';
+import pick from 'lodash/pick';
+
+export default /* @ngInject */ class XdslDiagnosticDetailsCtrl {
+  constructor($rootScope, $scope, $state, $translate) {
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/diagnostic/details/details.controller.js:6]:
```suggestion
  /* @ngInject */
  constructor($rootScope, $scope, $state, $translate) {
```

File: packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/diagnostic/diagnostic.controller.js [hunk 38]
```diff
@@ -0,0 +1,6 @@
+export default /* @ngInject */ class XdslDiagnosticCtrl {
+  constructor($stateParams) {
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/diagnostic/diagnostic.controller.js:2]:
```suggestion
  /* @ngInject */
  constructor($stateParams) {
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/activation/index.js [hunk 39]
```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import routing from './activation.routing';
+
+const moduleName = 'PackDomainActivation';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/activation/index.js:10]:
```suggestion
const moduleName = 'ovhManagerTelecomPackDomainActivation';
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/index.js [hunk 40]
```diff
@@ -0,0 +1,29 @@
...
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import controller from './domain.controller';
+import template from './domain.html';
+
+const moduleName = 'PackDomain';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/domain/index.js:11]:
```suggestion
const moduleName = 'ovhManagerTelecomPackDomain';
```

And for all `moduleName`.

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/add/index.js [hunk 41]
```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+import ngOvhTelecomUniverseComponents from '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import routing from './add.routing';
+
+const moduleName = 'PackEmailProAdd';
```
Comment (cbourgois) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/add/index.js:10]:
```suggestion
const moduleName = 'ovhManagerTelecomPackEmailProAdd';
```

**Review by antleblanc — APPROVED** [review]

**Pull request merged**